README.md

```diff
@@ -2,16 +2,14 @@
 
 This repository is used to create docker images that can be used for testing the DAOstack stack, for example in  `@daostack/client` and `@doastack/alchemy`
 
-# The test environment
-
 
 The test environment consists of 3 (related) docker images:
 
 * https://hub.docker.com/r/daostack/test-env : an image with ganache with a large number of deployed contracts
 * https://hub.docker.com/r/daostack/subgraph-ipfs : an ipfs image which contains the proposal data and the subgraph definition
 * https://hub.docker.com/r/daostack/subgraph-postgres : a postgres image to use with graph-node, that contains the data of a subgraph called `daostack` that indexes the contracts in the `test_env` image
 
-## Using the package
+## How to use the docker images created in this repository
 
 
 These setups look like this:
```

Comment:
Look like what?

Reply:
"Visit the subgraph at"?

Reply:
"tag and publish the docker contains to dockerhub"  How?

Reply:
"will create a rebuild the images and publish them on dockerlhub."  You mean that the shell performs the following steps for you?